docs/docs/about.md

```diff
@@ -12,6 +12,12 @@ If you are looking for Reanimated 1 docs [please follow this link](https://docs.
 
 :::
 
+:::caution
+
+Reanimated v2 supports `react-native` since version `0.62+`
```

Comment:
Kinda awkwardly worded for me. What do you think about:

"Reanimated v2 only supports react-native 0.62+."

Reply:
Sounds better 👍